model/paused_resource.go

```diff
@@ -22,6 +22,10 @@ func (app *Application) UpdateFromRequestPaused(req *ResourceEditPausedRequest)
 	lastCheckedAt := req.LastCheckedAt
 
 	if availabilityStatus != nil {
+		if _, ok := ValidAvailabilityStatuses[*req.AvailabilityStatus]; !ok {
```

Comment:
small thing but probably `availabilityStatus` could be used here instead of `req.AvailabilityStatus`

Reply:
Sure, makes more sense :+1:  Changing it now...